docs/user-guide/commands/argocd_proj_add-destination.md

```diff
@@ -5,7 +5,7 @@
 Add project destination
 
 ```
-argocd proj add-destination PROJECT SERVER/NAME NAMESPACE [flags]
+argocd proj add-destination PROJECT SERVER/NAME NAMESPACE SERVICEACCOUNT [flags]
```

Comment:
This change is not required. As we have a separate object DestinationServiceAccount under AppProject